### ISSUE TITLE
Deep learning

### ISSUE THREAD

**Issue by SWAN88** [SWAN88/paper_reading#11]
# Paper info
URL: https://doi.org/10.1038/nature14539
Journal: Nature
Year: 2015

# What is it about??

This review explains the math behind deep learning briefly in the introduction: backpropagation algorithm, as well as convolution neural networks (CNN) and recurrent neural networks (RNN) in more detail.

# What's next to read??

# Comments
Stochastic gradient descent (SGD) consists of showing the input vector for a few examples, computing the outputs and the errors, computing the average gradient for those examples, and adjusting the weights accordingly.
Generalization ability: ability to produce sensible answers n new inputs that it has never seen during training
